iogt_content_migration/management/commands/load_v1_db.py

```diff
@@ -1006,6 +1008,30 @@ def map_survey_thank_you_text(self, row):
 
         return json.dumps(v2_thank_you_text)
 
+    def map_survey_terms_and_conditions(self, row):
+        sql = f'select * ' \
+              f'from surveys_surveytermsconditions stc, surveys_molosurveypage msp, wagtailcore_page wcp ' \
+              f'where stc.page_id = msp.page_ptr_id ' \
+              f'and stc.terms_and_conditions_id = wcp.id ' \
+              f'and stc.page_id = {row["page_ptr_id"]}'
+
+        cur = self.db_query(sql)
+        v1_term_and_condition = cur.fetchone()
+        cur.close()
+        if v1_term_and_condition:
+            return json.dumps([
+                {
+                    "type": "page_button",
+                    "value": {
+                        "page": self.v1_to_v2_page_map[v1_term_and_condition["terms_and_conditions_id"]].id,
+                    },
+                },
+                {
+                    "type": "paragraph",
+                    "value": v1_term_and_condition["title"],
+                },
```

Comment:
I think the page button on its own is sufficient. It already has the title printed on it.

Reply:
removed paragraph

Reply:
We want to keep the paragraph type in the StreamField, so in v2 it's possible to add both page buttons and paragraphs. Just for the migration we don't need a paragraph block because the title is already in the page button.

Reply:
added back as a field only.